test/fixtures/tree-shaking/src/plugins/instrumented-as-pure-node.js

```diff
@@ -0,0 +1,8 @@
+import {createPlugin} from 'fusion-core';
+
+const PLUGIN = /*@__PURE__*/ createPlugin({
```

Comment:
What is the purpose of the instrumented as pure files?

Reply:
Just thought it would be good to have a test case for this, in case that folks want to instrument their own code.